docs/integrations/custom-indicator.md

```diff
@@ -0,0 +1,80 @@
+---
+id: custom-indicator
+title: Custom Indicator
+description: Create a customized indicator
+---
+
+## Overview
+**CustomIndicator** is a new type of an indicator which allows you to create customized Indicators.
+
+## CustomIndicator Class
+
+* The CustomIndicator can get any custom name for the indicator.
+* Unlike other indicators the CustomIndicator can have any parameters wanted, it can be passed by the params argument, which is a dictionary where the key is the parameter name and the value is the parameter's value.
```

Comment:
```suggestion
* Unlike other indicators the CustomIndicator can have any parameters wanted, it can be passed by the `data` argument, which is a dictionary where the key is the parameter name and the value is the parameter's value.
```

According to the table underneath.

Reply:
fixed